packages/tools/piw-utils-internal/src/widgets/__tests__/__snapshots__/Pagination.spec.tsx.snap

```diff
@@ -2,32 +2,47 @@
 
 exports[`Pagination renders the current page correctly with server side paging 1`] = `
 <div
+  aria-label="Pagination"
```

Comment:
Do we need an aria label "pagination" if the role is already "pagination"? 🤔

Reply:
Yes, same as core DG pagination.

Reply:
It can be translatable in the future.